apps/rtm/controllers/add-report.js

```diff
@@ -13,7 +13,6 @@ module.exports = class AddReportController extends Controller {
     var reports = req.sessionModel.get('report') || [];
```

Comment:
probably out of scope of this pr, but as part of all our ongoing work, we should try and update to ES6